apps/web/src/utils/utils.ts

```diff
@@ -26,3 +26,18 @@ export function parsePayload(payload: string) {
     return {};
   }
 }
+
+export function getCookies(cookieName: string): string | null {
```

Comment:
The repo includes js-cookies lib so you can use that. Having said that, eventually you shouldn't need it as you need `await Clerk.session.getToken()` to get the latest session token.

Reply:
it looks like `js-cookie` is just a dependency of some other library, we don't have it as direct dependency, so I thought I would avoid installing it for few lines of code which will get eventually changed anyway

Reply:
It's already used in web here https://github.com/novuhq/novu/blob/d9814af178ce4466329c1293575593e8e40d7928/apps/web/src/utils/cookies.ts#L1